lightbluetent/home.py

```diff
@@ -55,10 +63,16 @@ def home():
     if user.societies:
         user_societies = user.societies
 
-        return render_template("home/home.html", page_title="Home", user_societies=user_societies, crsid=crsid)
+        running_meetings = {}
+
+        for society in user.societies:
```

Comment:
There is some inconsistency here, `user_societies` is defined as an alias to `user.societies` but it is not used here. I think we should not create this alias just because it's the same length as `user.societies` and isn't any easier to type.